src/EnergyPlus/api/runtime.py

```diff
@@ -31,50 +31,54 @@ def __init__(self, api: cdll):
         self.api.issueText.argtypes = [c_char_p]
         self.api.issueText.restype = c_void_p
         self.py_progress_callback_type = CFUNCTYPE(c_void_p, c_int)
-        self.api.registerProgressCallback.argtypes = [self.py_progress_callback_type]
```

Comment:
Accompanying Python changes here for different functional forms and state management function changes.